resources/views/users/contestants.blade.php

```diff
@@ -0,0 +1,13 @@
+@extends('layouts.master')
+
+@section('main_content')
+
+    @include('layouts.header',[
+        'title' => 'Contestants'
+    ])
+
+    @if ($contestants->count())
+        @include('users.partials._table_users', ['users' => $contestants, 'role' => 'Contestants'])
+    @endif
+
+@stop
```

Comment:
Maybe this file should be `/users/contestants/index.blade.php`. Since we might have a `/users/contestants/show.blade.php`? Or maybe just getting too into the weeds? Otherwise have it be called `contestants_index.blade.php` so it's more obvious the index view, since it's the pattern we've used in similar naming.